opencog/nlp/learn/scm/mst-parser.scm

```diff
@@ -399,6 +399,47 @@
 
 	(close-port file-port)
 )
+
+(define-public (export-mi)
+"
+  Export the word-pairs fMI for
+  every possible pair in the corpus, to allow analysis
+  of the MI-based MST-parser
+
```

Comment:
It would be better if we did not place cheap ugly hacks in the main body of the code. Something like this really should go into a separate directory, where you are maintaining your personal research tools. I have been using the `nlp/learn/learn-lang-dairy/` directory for messy stuff like this.  If you feel you need to share this with others, then it should be placed in some collaborative tools directory. It doesn't belong in the main body of processing code.

Reply:
I put the offending code in a separate file, as suggested. I put this file in the nlp/learn/run folder... is that OK? It felt unnecessary to add a separate directory for only one file